### DISCUSSION TITLE
E49-900MBL-01

### DISCUSSION THREAD

**Discussion by kurt7676** [tbnobody/OpenDTU#897] [General]
Hello!
I bought E49-900MBL-01 for openDTU connection. 
Only pinout schema I found is:
![dtu](https://user-images.githubusercontent.com/121697529/236881179-9252b1f3-d70c-4b69-81d7-1d43d177adca.jpg)
Can you retranslate pins above to standard CMT2300A?

Thx a lot!
Kurt

**Comment by Floris039** [comment]
I found out that CLK = pin 12, CS = pin 8, FCS = pin 10 and SDIO = pin 13 on the CMT board

**Comment by kurt7676** [comment]
I looked carefully at the PCB and see that the CMT2300A is controlled by an STM8 series MCU.
On the PCB we have the pins coming from the ST8 chip. First you need to upload the control code for the ST8, and then modify the openDTU source code accordingly?

**Reply by storchalberto** [reply]
I have connected the board via usb to my computer. If i push a button, i receive:  #SEND: ping  so the morule is running.
I want to setup the module (assign the recv frequency) to receive data from a device. Where can i found the documentation of the serial commands?
